packages/backend/CLAUDE.md

```
@@ -0,0 +1,344 @@
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+
+# Convex guidelines + Clerk setup
+- documentation: https://docs.convex.dev/
+- extending convex with components:
+    - documentation: https://docs.convex.dev/components
+    - components list: https://www.convex.dev/components
+
+## Extras
+### Push Notifications
+If you are doing a push notification related to data stored in convex follow this guide https://www.convex.dev/components/push-notifications, the base is already setup in the project the rest needs to be setup
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)